.travis.yml

```diff
@@ -8,6 +8,8 @@ matrix:
       env: ARCH=arm64 LD=ld.lld-8
     - name: "ARCH=x86_64"
       env: ARCH=x86_64
+    - name: "ARCH=ppc64le"
+      env: ARCH=ppc64le
```

Comment:
Additionally, let's add an additional `REPO=linux-next` target `if: type = cron` below.

Reply:
Thanks. I also sorted the entries.

Reply:
did you push that change?